test/memory/TaskManagerTest.java

```diff
@@ -19,10 +28,10 @@ abstract class TaskManagerTest<T extends TaskManager> {
     List<Task> list = new ArrayList<>();
```

Comment:
У всех полей данного класса должен быть модификатор доступа `protected`
```suggestion
    protected List<Task> list = new ArrayList<>();
```